src/ipyradiant/visualization/cytoscape.py

```diff
@@ -97,11 +96,19 @@ def log_edge_clicks(self, edge):
 
     @T.observe("graph")
     def update_cyto_widget_graph(self, change):
-        for node in list(self.cyto_widget.graph.nodes):
-            self.cyto_widget.graph.remove_node(node)
-        if len(self.cyto_widget.graph.nodes) != 0:
-            with self.log:
-                print("Unexpected number of nodes remaining after graph cleared.")
+        try:
```

Comment:
@lukasnijhawan Can you take a look at this and make sure you understand what Nick changed.